### ISSUE TITLE
缺少文件，構建失敗

### ISSUE THREAD

**Issue by gwankyun** [rust-tw/book-tw#94]
```
2022-04-08 11:11:57 [INFO] (mdbook::book): Book building has started
2022-04-08 11:11:57 [ERROR] (mdbook::preprocess::links): Error updating "{{#rustdoc_include ../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs:here}}", Could not read file for link {{#rustdoc_include ../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs:here}} (C:\Users\kyun\Documents\GitHub\book-tw\src\../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs)
```

於Win11下。

**Comment by weihanglo** [comment]
Sorry I don't have any spare Windows machine at this moment, though I guess it might be a problem from mdbook. Could you share the version of mdbook installed in your system? Collect the output of `mdbook --version` is sufficient.

**Comment by gwankyun** [comment]
> 

mdbook v0.4.14

**Comment by gwankyun** [comment]
> Sorry I don't have any spare Windows machine at this moment, though I guess it might be a problem from mdbook. Could you share the version of mdbook installed in your system? Collect the output of `mdbook --version` is sufficient.

mdbook v0.4.14

**Comment by dieterplex** [comment]
Reproduced in powershell 7 at Win 10 pro 21H2. And similar result in cmd prompt.
```
PS > cargo install mdbook
    Updating crates.io index
  Downloaded mdbook v0.4.18
  Downloaded 1 crate (1.5 MB) in 1.41s
  Installing mdbook v0.4.18
...<skip>...
   Compiling mdbook v0.4.18
    Finished release [optimized] target(s) in 8m 59s
   Replacing C:\Users\arain\.cargo\bin\mdbook.exe
    Replaced package `mdbook v0.4.15` with `mdbook v0.4.18` (executable `mdbook.exe`)

PS > mkbook build
mkbook: The term 'mkbook' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

PS > ..\.cargo\bin\mdbook build
2022-06-05 21:53:12 [INFO] (mdbook::book): Book building has started
2022-06-05 21:53:13 [ERROR] (mdbook::preprocess::links): Error updating "{{#rustdoc_include ../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs:here}}", Could not read file for link {{#rustdoc_include ../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs:here}} (C:\Users\arain\book-tw\src\../listings/ch06-enums-and-pattern-matching/no-listing-15-binding-catchall/src/main.rs)
2022-06-05 21:53:13 [WARN] (mdbook::preprocess::links): Caused By: 系統找不到指定的路徑。 (os error 3)
2022-06-05 21:53:16 [INFO] (mdbook::book): Running the html backend
```

**Comment by dieterplex** [comment]
Fixed by #104
